From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kleis Auke Wolthuizen <github@kleisauke.nl>
Date: Mon, 10 Apr 2023 18:38:37 +0200
Subject: [PATCH 1/1] Revert "Enable GdkPixbuf loader for mingw as well"

This reverts commit 521d5c26bd86c7c103d1bbbf47aa55e30effad65.

Upstream-Status: Inappropriate [disable feature]
This patch is just for our convenience.

diff --git a/gdk-pixbuf/CMakeLists.txt b/gdk-pixbuf/CMakeLists.txt
index 1111111..2222222 100644
--- a/gdk-pixbuf/CMakeLists.txt
+++ b/gdk-pixbuf/CMakeLists.txt
@@ -1,4 +1,4 @@
-if(UNIX OR MINGW)
+if(UNIX)
   find_package(PkgConfig)
   pkg_check_modules(GDKPIXBUF2 gdk-pixbuf-2.0)
 
